server/src/main/kotlin/com/trib3/server/Extensions.kt

```diff
@@ -0,0 +1,15 @@
+package com.trib3.server
+
+/**
+ * Conditionally calls the specified function [block] with `this` value and returns its result,
+ * or returns `this` value if the [condition] is false.
+ *
+ * Useful for interacting with builder/fluent APIs.
+ */
+fun <T> T.runIf(condition: Boolean, block: T.() -> T): T {
```

Comment:
LOL I thought this was a new built in function when I saw it above. Very handy

Reply:
yeah, i wish we had a better place to put it, but don't feel like introducing a core/common module just for one function!